.github/CODEOWNERS

```diff
@@ -75,7 +75,7 @@
 /StRoot/StPass0CalibMaker                          @genevb            @iraklic
 /StRoot/StPico*                                    @nigmatkulov       @marrbnl     @plexoos
 /StRoot/StPxl*                                     @starsdong         @GuannanXie
-/StRoot/StRefMultCorr                              @nigmatkulov
+/StRoot/StRefMultCorr                              @nigmatkulov       @zsweger
```

Comment:
An invitation to join star-bnl has been sent to this user previously but was not accepted.

![Screen Shot 2022-08-17 at 12 30 03 PM](https://user-images.githubusercontent.com/5005079/185194278-5a53dc53-e4c2-4a5c-af08-eb105bbeca22.png)

I suggest someone who knows this user communicates directly in order to add as a code owner with write privileges

Reply:
How one can invite Zach to join the project?

Reply:
The invitation has been sent and is pending. @zsweger needs to follow the link in the email.

Reply:
I see that Zach has accepted the invitation and became a maintainer of /StRoot/StRefMultCorr